deployments/ocean/config/staging.yaml

```diff
@@ -1,7 +1,6 @@
 jupyterhub:
   proxy:
     https:
-      enabled: false
       hosts:
       - ocean.pangeo.io
```

Comment:
should this be `staging.ocean.pangeo.io`?

Reply:
@rabernat aha, yes it should be!